docs/UserGuide.md

```diff
@@ -1,196 +1,164 @@
-</h5>
+### Code Panel
+
+The `Code Panel` allows users to see the code attributed to a specific author. Click on the name of the author in the `Chart Panel` to display the `Code Panel` on the right.
+* The Code Panel shows the files that contain author's contributions, sorted by the number of lines written.
+* Clicking the file title toggles the file content.
+* Code attributed to the author is highlighted in green.
+* Non-trivial code segments that are not written by the selected author are hidden by default, but you can toggle them by clicking on the `...` icon.
+
 
-#### Git Author Name
-`Git Author Name` refers to the customizable author's display name set in the local `.gitconfig` file.
-It is displayed as author name as opposed to the `GitHub ID` in the entries.
-For example, in the Git Log's display:
+### Tool Bar
+The `Tool Bar` at the top provides a set of configuration options that control the Chart Panel.
+* `Search` : filters the author and repository by keywords.
+  * Multiple keywords/terms can be used, separated by spaces.
+  * Entries that contain _any_ (not necessarily _all_) of the search terms will be displayed.
+* `Sort by` : sorting criteria for the Chart Panel
+  * `Total Contribution` : the amount of lines, written by the author, in the repository.
+  * `Variance` : the [variance](https://en.wikipedia.org/wiki/Variance) of the number of lines that the author has contributed to the repository daily, sorted from low to high.
+  * `Author Name` : the author's display name.
+  * `Team Name` : see note [1] below.
+* `Granularity` : the period of time for which commits are aggregated in the Ramp Chart.
+    * `Day`: commits within a day (commits made within 00:00 to 23:59 in **GMT+8**) are shown as one ramp
+    * `Week`:  commits within a week are shown as one ramp
+* `Since`, `Until` : the date range for the Ramp Chart (not applied to the Contribution Bars).
+* `Reverse` : if checked, the sorting will be done in the reverse order of the default sorting order
+* `Group` : if checked, authors from the same team<sup>[1]</sup> will be grouped together. This has precedence over the `sort by` feature; the `sort by` feature will only change display order of the authors under the same team<sup>[1]</sup>.
+
+Notes:<br>
+[1] **`Team `**: the team name is constructed as `ORGANIZATION_REPOSITORY_BRANCH` e.g., `resposense_reposense_master`
+
+**Bookmarking a specific toolbar setting**: The URL changes according to the toolbar configuration. You can save a specific configuration of the report by bookmarking the url (using browser functionality).
+
+<hr>
+
+## Configuring a Repo to Provide Additional Data to RepoSense
+
+When a repo is being analyzed by RepoSense, there are **two ways repo owners can provide additional details to RepoSense**: using a config file, or annotating code using `@@author` tags. The two approaches are explained in the sections below.
+
+### Using a Config File
```

Comment:
This name conflicts with the Config (`.csv`) file below.
Perhaps name this back to 'Standalone Config'?

Reply:
The heading was meant to be read in conjunction with the previous heading, as in `Do X ... Using Y`. I added an `Option X:` in front to make it clearer. See if that works. I'm trying to minimize (and eventually eliminate/replace) the term `standalone config`. While the term uniquely identifies the file, is not intuitive on its own, especially given that the reader doesn't know about any other config file at this point.

Reply:
Adding `Option X:` in front makes it feel like we can only use 1 of them at a time, when actually we are allowed to mix them. 
For now, I think its good enough that we separate the 2 config with `json` and `csv` respectively.

Reply:
That's a good point. Will give it another try later tody.

Reply:
Updated. @yamidark